app/lib/github_badges.rb

```diff
@@ -37,7 +37,7 @@ def grant!
         end
 
         user_emails = {}
-        User.with_email(regular_emails).each do |user|
+        User.where(staged: false).with_email(regular_emails).each do |user|
```

Comment:
```suggestion
        User.real.where(staged: false).with_email(regular_emails).each do |user|
```

I a tiny bit better, though very unlikely to hit edge cases